doc/python/hover-text-and-formatting.md

```diff
@@ -138,6 +138,35 @@ fig = px.scatter(df_2007, x="gdpPercap", y="lifeExp", log_x=True,
 fig.show()
 ```
 
+### Disabling or customizing hover of columns in plotly express
```

Comment:
Since we're in the neighbourhood, could you please add a new section saying something like "PX uses `hovertemplate` to format its hovers, and you may disable or overwrite this with `.update_traces(hovertemplate='whatever')` and provide a short example showing the actual `hovertemplate` from `fig.data.[0]` and then showing how to change it?

Reply:
done